code/modules/organs/organ_description.dm

```diff
@@ -79,7 +79,7 @@
 	default_type = /obj/item/organ/external/head
 	default_bone_type = /obj/item/organ/internal/bone/head
 
-	max_damage = 75
+	max_damage = 90
```

Comment:
This will be very good for the economy

Reply:
For numbers, this will mean a bullet will require to reach
121 damage for a chance to tear off a head, and
135 damage for a chance to gib a head.

As opposed to the old numbers of
101 damage to tear off the head and
112 damage to gib the head.

Since the chance to gib is also reduced, this will make shooting the head a whole lot less lethal, and when you do shoot enough, it is more likely for the head to be torn off first before gib, making resusciation easier.